src/lib/utils/titleCase.ts

```
@@ -1,9 +1,20 @@
 export function toTitleCase(str: string): string {
     const minorWords = new Set(['a', 'an', 'the', 'and', 'but', 'or', 'for', 'nor', 'on', 'at', 'to', 'with', 'in']);
-    
+    const technicalTerms = new Set(['eip']);
     const romanNumeralPattern = /(^|\s|-)(I{1,3}|IV|V|VI{1,3}|IX|X)(\s|$|:)/i;
+    const hyphenatedEipPattern = /^(eip)-(\d+)$/i;
 
     return str.split(' ').map((word, index) => {
+        const eipMatch = word.match(hyphenatedEipPattern);
+        if (eipMatch) {
+            const [_, eipPart, numberPart] = eipMatch;
```

ESLint

[error] '_' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)